src/serverenvironment.cpp

```diff
@@ -867,6 +869,13 @@ class ABMHandler
 			}
 			aabm.check_required_neighbors = !required_neighbors_s.empty();
 
+			const std::vector<std::string> &without_neighbors_s =
+				abm->getWithoutNeighbors();
+			for (const std::string &without_neighbor_s : without_neighbors_s) {
+				ndef->getIds(without_neighbor_s, aabm.without_neighbors);
+			}
```

Comment:
I know you just copy-pasted this but the code is weirdly complicated for no reason
```suggestion
			for (const auto &name : abm->getWithoutNeighbors())
				ndef->getIds(name, aabm.without_neighbors);
```